pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -554,27 +554,16 @@ func (adc *attachDetachController) nodeAdd(logger klog.Logger, obj interface{})
 	if node == nil || !ok {
 		return
 	}
-	nodeName := types.NodeName(node.Name)
-	adc.nodeUpdate(logger, nil, obj)
-	// kubernetes/kubernetes/issues/37586
-	// This is to workaround the case when a node add causes to wipe out
-	// the attached volumes field. This function ensures that we sync with
-	// the actual status.
-	adc.actualStateOfWorld.SetNodeStatusUpdateNeeded(logger, nodeName)
-}
-
-func (adc *attachDetachController) nodeUpdate(logger klog.Logger, oldObj, newObj interface{}) {
-	node, ok := newObj.(*v1.Node)
-	// TODO: investigate if nodeName is empty then if we can return
```

Comment:
Is it possible that the name of a Node object from the informer is an empty string? 

cc @xing-yang